src/app/ui/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -50,14 +63,18 @@ const dropdownContentStyles = css({
   borderRadius: 'xs',
   boxShadow:
     '0px 12px 24px 0px rgba(18, 16, 15, 0.08), 0px 4px 8px 0px rgba(18, 16, 15, 0.08), 0px 0px 2px 0px rgba(18, 16, 15, 0.08)',
-  p: 'space.02',
+  p: '0',
   willChange: 'transform, opacity',
   zIndex: 999,
-  _closed: { animation: 'slideDownAndOut 140ms ease-in-out' },
-  _open: { animation: 'slideUpAndFade 140ms ease-in-out' },
+  // _closed: { animation: 'slideDownAndOut 140ms ease-in-out' },
```

Comment:
@kyranjamie - @fbwoolf helped me fix some tests by disabling these animations [here](https://github.com/leather-wallet/extension/pull/5012)

> The root problem though, @kyranjamie, are the changes you made to the content component? If I comment out the _open and _closed animations you added the menu stays closed during the tests and pass. If not, you can see when you run the tests locally in UI Mode using the --ui flag with Playwright, the settings menu was staying open and not allowing the button to be clicked? 

I'm not sure if we want to re-add them separately? We only caught them in this PR as it's actually using the dropdown menu in teses

Reply:
I gather the animations caused some difficulty with the tests here. But I don't share the conclusion that therefore the animations are bad and must go. Previously, they were applied to the wrong data attribute entirely.

This PR can leave them commented out, it isn't end of the world for them not to be animated for a short while. But the ambition should be to reintroduce them.

Reply:
OK thanks 👍 Mark also mentioned he likes the `fadein` animation we have on `HomeLayout` so we can investigate adding that elsewhere. 

I opened an issue: https://github.com/leather-wallet/extension/issues/5054

Reply:
@kyranjamie why do you say they were applied incorrectly when I followed the Radix example here: https://www.radix-ui.com/primitives/docs/components/dropdown-menu

Reply:
Didn't mean to say incorrectly, just mentioned because the close animation didn't work because it was applied only on [the one `data-side` property](https://github.com/leather-wallet/extension/commit/a82457d8f04e94b696fe4b54edbd8981f311c353#diff-ad9e50c4c62047dae395e6cf44f04e24c3972c111a43e8a80cba317aeda6b910R48-R50). The only change here is applying a new animation to the open/close properties instead.

Reply:
Ok, got it, I thought that was the close animation ...I need to understand those better.

Reply:
I think we do the same on our tooltip?